recipes/eo-learn/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "eo-learn" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: ecaf86b2e645bd28c091320029d784d0f6c4c9fd51c61eeaf491709f7c03ea8b
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - eo-learn-core 
```

Comment:
do you need all of those dependencies to install this package or just to run it?

Reply:
It is very likely that you don't need them. 

To install the software, all you need is to run the `setup.py` file, not the actual software dependencies.

Reply:
AFAIK this package doesn't contain a runnable, it's just a "wrapper" for all the sub packages and installing `eo-learn` is a faster way to install all the subpackages, which was the intention in putting all the sub packages as host requirements [eo-learn](https://pypi.org/project/eo-learn/)